stack.yaml

```diff
@@ -2,13 +2,13 @@
 # For more information, see: http://docs.haskellstack.org/en/stable/yaml_configuration.html
 
 # Specifies the GHC version and set of packages available (e.g., lts-3.5, nightly-2015-09-21, ghc-7.10.2)
-resolver: lts-5.1
+resolver: lts-5.9
 
 # Local packages, usually specified by relative directory name
 packages:
 - '.'
 # Packages to be pulled from upstream that are not in the resolver (e.g., acme-missiles-0.3)
-extra-deps: []
+extra-deps: [aeson-0.11.0.0]
```

Comment:
@TheJJ This is btw necessary for using aeson >= 0.11, because lts-5.9 provides aeson-0.9 by default and stack won't be able to pull in dependencies otherwise